lib/utils/schema.js

```diff
@@ -0,0 +1,76 @@
+import Joi from 'joi'
+
+const entrySchema = {
+  sys: Joi.object(),
+  fields: Joi.object()
+}
+
+const contentTypeSchema = {
+  sys: Joi.object(),
+  fields: Joi.array().required().items(Joi.object().keys({
+    id: Joi.string().required(),
+    name: Joi.string().required(),
+    type: Joi.string().required().regex(/Symbol|Text|Integer|Number|Date|Object|Boolean|Array|Link/),
+    validations: Joi.array(),
```

Comment:
Shouldn't we have a schema for validations?

Reply:
validations is so vague and it depends on the field. We can live without validating the validations for now, also we didn't have any issue related to the validations

Reply:
Alright, sounds good to me